chatkit-core/src/main/kotlin/com/pusher/chatkit/messages/MessageService.kt

```diff
@@ -1,78 +1,70 @@
 package com.pusher.chatkit.messages
 
-import com.pusher.chatkit.CustomData
 import com.pusher.chatkit.PlatformClient
-import com.pusher.chatkit.files.AttachmentBody
 import com.pusher.chatkit.files.DataAttachment
 import com.pusher.chatkit.files.FilesService
 import com.pusher.chatkit.files.GenericAttachment
 import com.pusher.chatkit.files.LinkAttachment
 import com.pusher.chatkit.files.NoAttachment
-import com.pusher.chatkit.messages.multipart.NewPart
-import com.pusher.chatkit.messages.multipart.UrlRefresher
-import com.pusher.chatkit.messages.multipart.request.AttachmentId
-import com.pusher.chatkit.messages.multipart.request.Part
-import com.pusher.chatkit.messages.multipart.upgradeMessageV3
+import com.pusher.chatkit.files.api.AttachmentBodyApiType
 import com.pusher.chatkit.rooms.RoomService
-import com.pusher.chatkit.rooms.V3MessageBody
 import com.pusher.chatkit.users.UserService
-import com.pusher.chatkit.util.parseAs
 import com.pusher.chatkit.util.toJson
 import com.pusher.util.Result
 import com.pusher.util.asSuccess
-import com.pusher.util.collect
 import elements.Error
-import elements.Errors
-import java.io.ByteArrayOutputStream
-import java.net.URLEncoder
 
 internal class MessageService(
     private val legacyV2client: PlatformClient,
     private val client: PlatformClient,
     private val userService: UserService,
     private val roomService: RoomService,
-    private val urlRefresher: UrlRefresher,
+//    private val urlRefresher: UrlRefresher,
     private val filesService: FilesService
 ) {
+    @Suppress("UNUSED_PARAMETER")
     fun fetchMessages(
         roomId: String,
         limit: Int,
         initialId: Int?,
         direction: Direction
-    ): Result<List<Message>, Error> =
-            fetchMessagesParams(limit, initialId, direction).let { params ->
-                legacyV2client.doGet<List<Message>>("/rooms/$roomId/messages$params").flatMap { messages ->
-                    messages.map { message ->
-                        userService.fetchUserBy(message.userId).map { user ->
-                            message.user = user
-                            message
-                        }
-                    }.collect().mapFailure { errors ->
-                        Errors.compose(errors)
-                    }
-                }
-            }
+    ) {}
+//            : Result<List<Message>, Error> =
+//            fetchMessagesParams(limit, initialId, direction).let { params ->
+//                legacyV2client.doGet<List<Message>>("/rooms/$roomId/messages$params").flatMap { messages ->
+//                    messages.map { message ->
+//                        userService.fetchUserBy(message.userId).map { user ->
+//                            message.user = user
```

Comment:
this user enrichment part shall go as it will be different in v2 but the rest is still relevant